ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/BaseApplicationResponseJvm.kt

```diff
@@ -297,18 +300,19 @@ public actual abstract class BaseApplicationResponse actual constructor(
          * to start response object processing via [respondOutgoingContent]
          */
         public actual fun setupSendPipeline(sendPipeline: ApplicationSendPipeline) {
-            sendPipeline.intercept(ApplicationSendPipeline.Engine) { response ->
-                if (response !is OutgoingContent) {
+            sendPipeline.intercept(ApplicationSendPipeline.Engine) { body ->
+                if (call.isHandled) return@intercept
```

Comment:
is this change relevant?

Reply:
Yep, it's a part of fixing how new plugin API works.